src/sentry/runner/commands/config.py

```diff
@@ -92,3 +92,13 @@ def generate_secret_key():
     "Generate a new cryptographically secure secret key value."
     from sentry.runner.settings import generate_secret_key
     click.echo(generate_secret_key())
+
+
+@config.command()
+@configuration
```

Comment:
I guess you mean the `@configuration` decorator?  Removing this is fine, but removing `@config.command()` means the runner doesn't pick it up.